Apps/Src/Current.c

```diff
@@ -108,10 +107,27 @@ void Task_AmperesMonitor(void *p_arg) {
 
     OS_ERR err;
 
+	bool amperesHasBeenChecked = false;
+
     while(1) {
         // BLOCKING =====================
         // Update Amperes Measurements
+		Current_UpdateMeasurements();
+
         // Check if amperes is NOT safe:
+		SafetyStatus amperesStatus = Current_CheckStatus(true);
```

Comment:
If we are using Voltage.c to catch the case where we are discharging during an under-voltage and for this to check the full range of currents, we should pass `false` to this instead of `true`.